test/unit/helper.spec.ts

```diff
@@ -0,0 +1,20 @@
+import {expect} from 'chai';
+import {format} from '../../src/commandHelper';
+
+describe('Command Helper', () => {
+  it('should format text with indents', () => {
+    const message = 'message to format';
+    const options = {
+      indent: 2,
+    };
+    let formattedMessage = format(message, options);
+    expect(formattedMessage).to.equal('  ' + message, 'adds 2 spaces');
+
+    options.indent = -1;
+    formattedMessage = format(message, options);
+    expect(formattedMessage).to.equal(
+      message,
+      'no spaces added if indent number is < 0'
+    );
```

Comment:
In general, I'd recommend testing for specific conditions like this into their own test case, ex: `it('should not add spaces with a negative indent')` or something